modules/deploy/version.yml

```diff
@@ -1,6 +1,4 @@
 # This file is generated by https://github.com/gpii-ops/gpii-version-updater.
 ---
-
-flowmanager: "gpii/universal@sha256:e99862c916b72e9ad1801f572b8ba0a619bac55cc0d7eb4cc446633a289f5962"
-preferences-dataloader: "gpii/preferences-dataloader@sha256:0156db0e8ae5c919f8b3e6a5c15b485b9a221731e213bbb073c8697ac70d7417"
-preferences: "gpii/universal@sha256:e99862c916b72e9ad1801f572b8ba0a619bac55cc0d7eb4cc446633a289f5962"
+flowmanager: "cindyqili/vagrant-universal@sha256:383f39aab431e5d657724ce00877a69c44447136a46fc626448a3dcba68192b8"
+gpii-dataloader: "cindyqili/gpii-dataloader@sha256:f5a343801cb8bfaf9bf2d46ae01d010a6c4753de61c464c70a89f26020a3e650"
```

Comment:
I guess that the source of the all images should be the official `gpii` account. Are your changes merged in the master branch of universal?, do the containers `cindyqili/vagrant-universal` and `cindyqili/gpii-dataloader` have any additional changes that we should consider to include in the master branch of the GPII's containers?

Reply:
Thanks for looking into this pull request, @amatas.

The containers here are built upon [my GPII-2630 universal branch](https://github.com/cindyli/universal/tree/GPII-2630) instead of the universal master. The changes made in GPII-2630 branch that will affect the new containers are described in [the universal GPII-2630 pull request](https://github.com/GPII/universal/pull/591). These lines are now pointing to new containers in my docker hub to ensure that proper containers can be pulled for testing this pull request.

I believe at runtime, this file (version.yml) will be generated and overwritten by gpii-ops/gpii-version-updater, which has [another pull request](https://github.com/gpii-ops/gpii-version-updater/pull/2) issued for the purpose. But I do agree in the master, these lines should point to GPII docker hub instead of a developer's to reduce confusion.

I'm thinking when the universal GPII-2630 pull request is merged into the master, and at then GPII docker hub does contain proper containers that work with this pull request, these 2 lines can be updated to point to GPII docker hub before merging it into the master.

Please let me know if I should change these lines now.

Reply:
Deploying Cindy's GPII-2630 work will require a lot of coordination (PRs in multiple repos, some manual steps to handle `preferences` going away). Plus, as Cindy notes, the file is generated dynamically so it should end up with the correct contents once everything is merged and deployed.

Hence, I'm not worried about the contents of `version.yml` in this branch. I think we can leave it alone and verify its correctness later.